deployment/aws-waf-security-automations-firehose-athena.template

```diff
@@ -408,7 +406,7 @@ Resources:
       Type: AWS::Athena::WorkGroup
       Condition: AthenaLogParser
       Properties:
-        Name: !Join ['-', ['WAFAddPartitionAthenaQueryWorkGroup', !Ref UUID]]
+        Name: !Join ['-', [!Ref ParentStackName, 'WAFAddPartitionAthenaQueryWorkGroup']]
```

Comment:
Using the stack name instead of UUID, otherwise all the resource get updated each time you do a stack update